Header header
Point2D direction
Size2D sigma
